### PR TITLE
TFF 77: Invalid student id

### PR DESCRIPTION
**Pull request by thofmann**
### Summary

Prompts an error modal when an invalid/missing student id is scanned.

Note: I think the PR should work as-is, but the modal won't appear until TFF-71 is completed and merged. We should probably wait until that story is merged into dev before this PR gets merged so that we can verify it actually works.

This also requires the [Sister PR to consus](https://github.com/TheFourFifths/consus/pull/16) to work.

### Checklist

- [x] Did you run `npm test`?
- [x] Did you update/add documentation for new methods or changed functionality?

Finally, is it ready to be reviewed and merged: yes :frog: 

### How to Review

1. Run the server locally, using the changes from the [sister PR](https://github.com/TheFourFifths/consus/pull/16).
2. Start the client.
3. Enter a 6-digit id that does not correlate to a student yet (e.g. 999999).
4. Verify that a modal pops up with an appropriate error message.

### Links

- [TFF-77](https://msoese.atlassian.net/browse/TFF-77)
- [Sister PR to consus](https://github.com/TheFourFifths/consus/pull/16)

### CODE REVIEW

File: test/unit/store/cart-store.js [hunk 1]
```diff
@@ -3,6 +3,9 @@ import CartStore from '../../../.dist/store/cart-store';
 import { assert } from 'chai';
 
 describe('CartStore', () => {
+    before(() => {
+        return Dispatcher.handleAction('CLEAR_ALL_DATA');
```
Comment (tondef001) [inline test/unit/store/cart-store.js]: 👍

File: test/unit/store/cart-store.js [hunk 2]
```diff
@@ -30,7 +33,6 @@ describe('CartStore', () => {
         assert.strictEqual(CartStore.getItems()[0].address, '123');
         Dispatcher.handleAction('CHECKOUT_SUCCESS');
         assert.strictEqual(CartStore.getItems().length,0);
-        Dispatcher.handleAction('NO_STUDENT_FOUND');
     });
 
     it('should clear items on cancel', () => {
```
Comment (tondef001) [inline test/unit/store/cart-store.js]: probably should delete the NO_STUDENT_FOUND action in this test too

**Comment by gearedmonkey** [comment]
mhm

File: test/unit/store/cart-store.js (hunk 2 above)
Comment (thofmann) [inline test/unit/store/cart-store.js]: Good catch, fixed.

**Pull request merged**